src/sentry/models/sentryappwebhookerror.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import EncryptedJsonField, FlexibleForeignKey, Model
+
+
+class SentryAppWebhookError(Model):
+    __core__ = False
+
+    date_added = models.DateTimeField(default=timezone.now)
+
+    sentry_app = FlexibleForeignKey("sentry.SentryApp", related_name="webhook_errors")
+
+    organization = FlexibleForeignKey(
+        "sentry.Organization", related_name="sentry_app_webhook_errors"
+    )
+
+    # If there is a related Sentry error, store it here
+    error_id = models.CharField(max_length=64, null=True)
+
+    request_body = EncryptedJsonField()
+
+    request_headers = EncryptedJsonField()
+
+    event_type = models.CharField(max_length=64)
+
+    # We need to store this rather than just fetch it from the related sentry app in case the URL is changed
+    webhook_url = models.URLField()
+
+    response_body = models.TextField()
```

Comment:
Same as `request_body`.

Reply:
@scefali  @mnoble  I'm actually not sure about this one. Right now I'm storing this field as `body = safe_urlread(resp)` where resp is an http response but I don't know what the type of `resp.content` is supposed to be -- is there a way to standardize it? In the codebase it seems like it's usually `json.loads(body)` but it seems that the content could sometimes be a string already?

Reply:
@nolachen I would recommend trying it out with a test integration and see what happens.

Reply:
@scefali so far what i've tried returns a string. and it seems like from https://docs.djangoproject.com/en/1.8/ref/request-response/#django.http.HttpResponse.content that it should always be a string? In that case is any conversion necessary?